### ISSUE TITLE
Dubious: Starting paragraphs with 'But'

### ISSUE THREAD

**Issue by cpitclaudel** [amperser/proselint#605]
Is there a source for the following warning?

```
misc.but No paragraph should start with a 'But'.
```

Here's what Garner says on the subject (but, 3rd edition, p 121):

> It is a gross canard that beginning a sentence with _but_ is stylistically slipshod. In fact, doing so is desirable in any number of contexts, as countless stylebooks have said.

**Comment by mcepl** [comment]
Moreover, there seems to be some problems with parsing (at least in
``rST``):

```rst
Ideological struggle
    In the conditions of freedom it is just a contest of ideas,
    but in the militant ideological state it means pushing of one
    ideology and suppressing all others. Weakness: what is
    imposed generates resistance.
```

Whatever one thinks about this particular problem, there is no start of the paragraph here.

Also, this is effectively a duplicate of the discussion on #98.

**Comment by sobjornstad** [comment]
I don't think I've ever tried to start a paragraph with a “but” (though I have no opinion on whether it should be considered wrong: sentences are fine, why not paragraphs?). But (haha) when using [semantic linebreaks](https://sembr.org) or indeed any kind of hard line breaks in your text, this rule yields false positives all over the place...it seems to be checking the beginning of *lines* instead of the beginning of paragraphs.

**Comment by Nytelife26** [comment]
> Here's what Garner says on the subject (but, 3rd edition, p 121):
> 
> > It is a gross canard that beginning a sentence with _but_ is stylistically slipshod. In fact, doing so is desirable in any number of contexts, as countless stylebooks have said.

That is referring to sentences, not paragraphs. Do note that in the [original issue] that lead to the creation of this rule, the subject of paragraphs and self-containment is addressed.

[original issue]: https://github.com/amperser/proselint/issues/98

**Comment by sanjayankur31** [comment]
> Moreover, there seems to be some problems with parsing (at least in `rST`):
> 
> ```
> Ideological struggle
>     In the conditions of freedom it is just a contest of ideas,
>     but in the militant ideological state it means pushing of one
>     ideology and suppressing all others. Weakness: what is
>     imposed generates resistance.
> ```
> 
> Whatever one thinks about this particular problem, there is no start of the paragraph here.


This also affects TeX/LaTeX. I filed a new issue for this: https://github.com/amperser/proselint/issues/1376

**Comment by Nytelife26** [comment]
While the parsing of paragraphs remains to be addressed in #1376, much of the guidance around the topic deals with the beginning of sentences. In general, modern English permits starting a sentence with a conjunction - informally, at least - to allow for clarity and flow. Paragraphs, on the other hand, should remain self-contained.
